dts/carrier-channel-ids.dtsi

```diff
@@ -0,0 +1,273 @@
+/* Copyright (c) 2013, The Linux Foundation. All rights reserved.
```

Comment:
Rename this file to something like `(msm8916-)?motorola-carrier-channel-ids.dtsi`